frontend/packages/console-dynamic-plugin-sdk/src/api/internal-api.ts

```diff
@@ -10,6 +10,7 @@ import {
   ResourceInventoryItemProps,
   DetailItemProps,
   DetailsBodyProps,
+  DroppableEditYAMLProps,
```

Comment:
Our main vendor bundle size almost doubles with this PR. I believe this import is the cause.